ACE/debian/cmp-autoconf-regular.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
This file shouldn't be needed anymore as far as I remember, if so, please remove it

Reply:
I removed this and `ACE/debian/update_file_versions.sh`, which are in the [Debian source repository](https://anonscm.debian.org/cgit/pkg-ace/pkg-ace.git/) but are not needed.